openassessment/__init__.py

```diff
@@ -2,4 +2,4 @@
 Initialization Information for Open Assessment Module
 """
 
-__version__ = '6.12.3'
+__version__ = '7.0.0'
```

Comment:
I don't think we need a major version update for this, since there aren't any backwards-incompatible changes.
Just `6.13.0` will do.